components/camera/transformpipeline/pipeline.go

```diff
@@ -67,6 +68,16 @@ type transformConfig struct {
 	Pipeline             []Transformation                   `json:"pipeline"`
 }
 
+// Validate ensures all parts of the config are valid.
+func (cfg *transformConfig) Validate(path string) ([]string, error) {
+	var deps []string
+	if len(cfg.Source) == 0 {
+		return nil, rutils.NewConfigValidationFieldRequiredError("%q is required", "source")
```

Comment:
the first string in the argument should just be the "path" variable from the Validate argument. 
```
rutils.NewConfigValidationFieldRequiredError(path, "source")
```